src/Symfony/Component/Encryption/Exception/DecryptionException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption\Exception;
+
+/**
+ * Thrown when a message cannot be decrypted.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+class DecryptionException extends \RuntimeException implements ExceptionInterface
+{
+    public function __construct($message = null, \Throwable $previous = null)
+    {
+        parent::__construct($message ?? 'Could not decrypt the ciphertext.', 0, $previous);
```

Comment:
```suggestion
        parent::__construct($message ?? 'Could not decrypt the cipher text.', 0, $previous);
```

Reply:
It bugs be too, but it is "ciphertext". 
https://en.wikipedia.org/wiki/Ciphertext

Reply:
@Nyholm ha thanks, maybe then change inside Exception classes, you wrote `cipher text` :)

Reply:
oh =)

Sure.